### DISCUSSION TITLE
Discussion: New website design

### DISCUSSION THREAD

**Discussion by dan-is-not-the-man** [monero-project/monero-site#2372] [General]
Creating this because it was becoming Post malone in the matrix website room.Here is a list of current examples for new website design. If you have any designs or demo websites you want to submit please comment:

1. [Janaka design and demo site](https://getmonero-unofficial-beta.vercel.app)

2.  [Diego's design](https://www.figma.com/file/yT4JTUfvBkAsBDPNPJL1gx/Monero-Redesign-Concept?node-id=0%3A1)

3. [Hammermann09 design](https://rimgo.hostux.net/K7Cb9cD.png)

To contribute to discussion please join via:

- [Matrix room - #monero-site:monero.social](https://matrix.to/#/%23monero-site:monero.social)

- [Libera.chat, #monero-site](https://libera.chat/)

**Comment by rottenwheel** [comment]
Non-gradient, orangified Diego's version: https://qu.ax/OLtM.png

**Comment by fajrul352** [comment]
can i join this project?
